lib/api/redfish-1.0/update-service.js

```diff
@@ -0,0 +1,96 @@
+// Copyright © 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var redfish = injector.get('Http.Api.Services.Redfish');
+var Promise = injector.get('Promise'); // jshint ignore:line
+var _ = injector.get('_');  // jshint ignore:line
+var controller = injector.get('Http.Services.Swagger').controller;
+var Errors = injector.get('Errors');
+var workflow = injector.get('Http.Services.Api.Workflows');
+var waterline = injector.get('Services.Waterline');
+
+var updateFirmware = controller({success: 201}, function (req, res) {
+    var payload = req.swagger.params.payload.value;
+    var targetNodeIds = payload.Targets;
+    var errorList = [];
+    var transactionError = false;
+    var options = redfish.makeOptions(req, res, payload);
+
+    return Promise.try(function () {
```

Comment:
Since there is only `Promise.map` within the `Promise.try`, so Promise.try is not necessary.